raiden/raiden_service.py

```diff
@@ -212,14 +220,41 @@ def update_monitoring_service_from_balance_proof(
     if raiden.config['services']['monitoring_enabled'] is False:
         return
 
+    channel_state = views.get_channelstate_by_canonical_identifier(
+        chain_state=views.state_from_raiden(raiden),
+        canonical_identifier=CanonicalIdentifier(
+            chain_identifier=raiden.chain.network_id,
+            token_network_address=new_balance_proof.token_network_identifier,
+            channel_identifier=new_balance_proof.channel_identifier,
+        ),
+    )
+
+    msg = (
+        f'Failed update monitoring service due to failing to find '
+        f'channel: {new_balance_proof.channel_identifier} '
+        f'token_network_address: {pex(new_balance_proof.token_network_identifier)}.'
+    )
+    assert channel_state, msg
+
+    balance = channel.get_balance(
+        sender=channel_state.our_state,
+        receiver=channel_state.partner_state,
+    )
+
+    if balance < MONITORING_MIN_CAPACITY:
```

Comment:
Shouldn't this error be shown to the user, as monitoring is enabled but cannot happen due to this preconditions failing.

Reply:
+1 This is definitely a situation that requires user action

Reply:
Done